prow/cmd/deck/main.go

```diff
@@ -153,8 +155,8 @@ func (o *options) Validate() error {
 		}
 	}
 
-	if o.hiddenOnly && o.showHidden {
-		return errors.New("'--hidden-only' and '--show-hidden' are mutually exclusive, the first one shows only hidden job, the second one shows both hidden and non-hidden jobs")
+	if o.hiddenOnly && o.showHidden || o.tenantIDs.Strings() != nil && (o.hiddenOnly || o.showHidden) {
```

Comment:
I had trouble understand this without more parentheses, how about

```suggestion
	if (o.hiddenOnly && o.showHidden) || (o.tenantIDs.Strings() != nil && (o.hiddenOnly || o.showHidden)) {
```

Reply:
Done